rust/stackable-cockpitd/src/main.rs

```diff
@@ -23,6 +25,18 @@ mod cli;
 async fn main() -> Result<(), Whatever> {
     let cli = Cli::parse();
 
+    // Construct the tracing subscriber
+    let format = fmt::format()
+        .with_level(false)
+        .with_ansi(true)
+        .without_time();
+
+    tracing_subscriber::fmt()
+        .with_max_level(LevelFilter::INFO)
```

Comment:
This should be controlled by envvar instead

Reply:
Switched it. But not it does not show by default :/ Should we change the default to Info?

Reply:
I'd say so. It can probably be useful to look at [how op-rs handles this](https://github.com/stackabletech/operator-rs/blob/c234cef5afa2e4624d6ed21d013a7a4f805c2e11/src/logging/mod.rs).

Reply:
You can also look at this: https://github.com/stackabletech/stackable-cockpit/blob/main/rust/stackablectl/src/main.rs#L57-L61

Reply:
Yep, I'd say that's nicer. We might also want to convert op-rs to use the same at some point?